.ci/oci-e2e-deployment.sh

```diff
@@ -59,15 +59,15 @@ function createQuayPullSecrets() {
 function waitAppStudioToBeReady() {
     while [ "$(kubectl get applications.argoproj.io ${APPLICATION_NAME} -n ${APPLICATION_NAMESPACE} -o jsonpath='{.status.health.status}')" != "Healthy" ] ||
           [ "$(kubectl get applications.argoproj.io ${APPLICATION_NAME} -n ${APPLICATION_NAMESPACE} -o jsonpath='{.status.sync.status}')" != "Synced" ]; do
-        sleep 1m
+        sleep 2m
```

Comment:
this is not increasing timeout, just sleep

Reply:
It's on line 109 -> `timeout --foreground 10m bash -c waitBuildToBeReady`

Reply:
Lol, my mistake.